packages/stores/src/mockStore.ts

```diff
@@ -90,7 +104,7 @@ const mock = (): TStores => {
                     needs_verification: [],
                     ownership: {
                         requests: [],
-                        status: 'none',
+                        status: 'verified',
```

Comment:
@yaswanth-deriv Please don't change the default value as it will affect test files 🙏🏻